.github/workflows/prod_deploy.yaml

```diff
@@ -0,0 +1,22 @@
+name: Deploy Package
+on:
+  release:
+    types: [published]
```

Comment:
what happens if we need to unpublish a release? i.e. if we spot something dodgy...
do we just manually roll back on PyPI?

Reply:
Can't roll back - once live on Pypi - you have to yank it. Then release a post-release:
https://snarky.ca/what-to-do-when-you-botch-a-release-on-pypi/

Unfortunately once live on PyPI, the same version number can't be used again. We'll have to suffix with `post1` etc. Hence I created a dummy github action for the release branch to test publishing to TestPYPI as a final 'line of defence' before pushing stuff to prod.

Still thinking about it, feels like an overkill at this stage, so not quite set in stone.